slam-libraries/viam-cartographer/src/slam_service/slam_service.cc

```diff
@@ -128,9 +144,8 @@ void SLAMServiceImpl::CreateMap() {
 
     if (this->starting_scan_number < 0 ||
         this->starting_scan_number >= int(file_list.size())) {
-        std::cerr << "starting_scan_number is out of bounds: "
-                  << this->starting_scan_number << std::endl;
-        return;
+        throw std::runtime_error("starting_scan_number is out of bounds: " +
+                                 std::to_string(this->starting_scan_number));
     }
 
     std::cout << "Beginning to add data....\n";
```

Comment:
Should we remove log.txt? Or keep it until we've implemented `GetPosition`?

Reply:
I think there's no harm in keeping it until then. Might come in handy for debugging?

Reply:
sounds good!